iceoryx_hoofs/testing/include/iceoryx_hoofs/testing/mocks/time_mock.hpp

```diff
@@ -40,15 +40,9 @@ class time_MOCK
     static bool doUseMock; // = true;
 };
 
-#if defined(QNX) || defined(QNX__) || defined(__QNX__)
 int clock_getres(clockid_t clk_id, struct timespec* res);
 int clock_gettime(clockid_t clk_id, struct timespec* res);
 int clock_settime(clockid_t clk_id, const struct timespec* res);
-#else
-int clock_getres(clockid_t clk_id, struct timespec* res) noexcept;
```

Comment:
Hint: `noexcept` version removed since it does not make sense at all to mock a c function and add a exception specifier which is not present in the original. This will lead to compilation errors like for instance in QNX or FreeBSD and are just wrong in linux and worked by accident.